torch/distributed/rpc/backend_registry.py

```diff
@@ -23,6 +23,17 @@ def _backend_type_repr(self):
 BackendType = enum.Enum(value="BackendType", names={})
 BackendType.__repr__ = _backend_type_repr
 
+def backend_registered(backend_name):
+    """
+    Checks if backend_name is registered as an RPC backend.
+
+    Arguments:
+        backend_name (str): string to identify the RPC backend.
+    Returns:
+        True if the backend has been registered with `register_backend`, else False.
```

Comment:
nit: double quote for `register_backend`

BTW, is this a user-facing API?

Reply:
It's not intended to by used by users, and we don't export this function anywhere in the RPC docs.